test/jasmine/tests/shapes_test.js

```diff
@@ -282,3 +285,399 @@ describe('Test shapes: a plot with shapes and an overlaid axis', function() {
         Plotly.plot(gd, data, layout).then(done);
     });
 });
+
+describe('Test shapes', function() {
+    'use strict';
+
+    var gd, data, layout, config;
+
+    beforeEach(function() {
+        gd = createGraphDiv();
+        data = [{}];
+        layout = {};
+        config = {
+            editable: true,
```

Comment:
Yep. This is the way to go. The other `editable: true` are currently in [`config_test.js`](https://github.com/plotly/plotly.js/blob/3c85d9eb47ce08177d951065fd66a7d9f9671e57/test/jasmine/tests/config_test.js#L45), but scoping them by the component(s) they affect is much better. Thanks!